### ISSUE TITLE
"Overview" should be editable on communities page

### ISSUE THREAD

**Issue by AllhandsNoFeet** [Couchers-org/couchers#1608]
Currently you can only edit the "General information" section under "Local Info" on the communities page.

You should also be able to edit the "Overview" field on the main community landing page.

This should have a character limit so it doesn't get too long (it's just supposed to be a very brief overview!) See example below:
![image](https://user-images.githubusercontent.com/79384268/125149082-653fea00-e0fc-11eb-8a3a-67aa7b4b217f.png)

I think a 500 character limit should be good, and there should be a "Learn more..." link at the bottom of the text that directs to the "local Info" section.

**Comment by aapeliv** [comment]
This is very low priority since there's only a few communities and it's a bit of an annoying thing to impement in the backend.

**Comment by AllhandsNoFeet** [comment]
@aapeliv I would argue this should have a higher priority since it's very visible. In fact, it's the first thing people see when they click on a community! Right now it looks super bare. Seems like fairly good value to make it editable too since we plan to build out communities and push them a bit more soon.

**Comment by aapeliv** [comment]
It's low priority-effort ratio.

It's part of the community building first steps: to submit a description, and Itsi will set it when creating the community. He can also set it at any time later.

**Comment by aapeliv** [comment]
@harshadbhere: how about this one?

**Comment by harshadbhere** [comment]
@aapeliv I will try to stick with backend as I am not versed with frontend.

**Comment by aapeliv** [comment]
This requires adding a backend API call to edit the description. I was intending for you to only do that bit and we’ll get a fronted dev to do the rest!

**Comment by harshadbhere** [comment]
Ok then, I will try it!

**Comment by harshadbhere** [comment]
@aapeliv is there any other way of communication like Slack? I had some queries which require your advice.

**Comment by aapeliv** [comment]
Yes; email me at "aapeli" at our domain and we can set up a time to meet and get you on slack!

**Comment by aapeliv** [comment]
Hi @harshadbhere: how're you doing on this? Do you need any help?
